Demo/Tests/MoyaProviderSpec.swift

```diff
@@ -348,6 +348,65 @@ class MoyaProviderSpec: QuickSpec {
                 expect(executed).to(beTruthy())
             }
         }
+
+        describe("a provider with custom sample response closures") {
+            it("returns sample data") {
+                let endpointResolution: MoyaProvider<GitHub>.EndpointClosure = { target in
+                    let url = target.baseURL.appendingPathComponent(target.path).absoluteString
+                    return Endpoint(URL: url, sampleResponseClosure: {.networkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+                }
+                let provider = MoyaProvider<GitHub>(endpointClosure: endpointResolution, stubClosure: MoyaProvider.ImmediatelyStub)
```

Comment:
Why the `MoyaProvider.ImmediatelyStub` instead of just `.ImmediatelyStub`? And have these not been updated to lowercase enums yet?